README.md

```diff
@@ -13,7 +13,7 @@ Our method for structuring Blocks and Elements doesn’t actually require any sp
   const Button = styled.button``;
 
   // Define your Icon styled component (the Element)
-  const Icon = styled(FontAwesome);
+  const Icon = styled(FontAwesome)``;
```

Comment:
Is the template tag necessary if it is empty?

Reply:
I think so. Otherwise it is the function that creates the styled component, not an instance of a styled component.